app/decorators/member_decorator.rb

```diff
@@ -0,0 +1,17 @@
+class MemberDecorator < SimpleDelegator
```

Comment:
We need to decorate a single field on the member, a whole decorator fills like overkill but oh well.